pact/exchange.repl

```diff
@@ -15,17 +15,28 @@
 (env-keys ["operate"])
 
 (load "ns.pact")
+(load "fungible-util.pact")
 (load "tokens.pact")
 (load "swap-callable.pact")
 (load "exchange.pact")
 
 
 (define-namespace 'test (sig-keyset) (sig-keyset))
-(load "test/fungible-util.pact")
 (load "test/ABC.pact")
 (load "test/XYZ.pact")
 (commit-tx)
 
+(begin-tx)
+(load "test/tokens-test-fungible.pact")
+(load "test/fungible.repl")
+(test-capability (swap.tokens.CREDIT TOKEN FUNDER_ACCT))
+(expect "fund success" true
+  (test-fund FUNDER_ACCT FUNDER_GUARD FUNDER_BALANCE))
+(commit-tx)
+
+(fungible-v2-test.suite tokens-test-fungible
+   fungible-test-helper-default "transfer-crosschain-tests")
```

Comment:
Wasn't really sure how this passed. It does show true in` exchange.repl`, but when I add another line that tests "rotate-tests" like this ,
```
(fungible-v2-test.suite tokens-test-fungible
   fungible-test-helper-default "rotate-tests")
```

 it shows error message like below in fungible.repl. It still shows true in exchange.repl. When I erase the extra test, then the error in fungible.repl disappears. 

![Screen Shot 2020-12-22 at 3 24 37 PM](https://user-images.githubusercontent.com/31594593/102930129-d7efbd80-4469-11eb-8796-08d736c9a615.png)

Reply:
Yeah, that's because `expect-XXX` errors don't actually halt execution, so the test returns true. The actual `pact` output returns "Load Failed" at the end but I don't think Atom does anything with that.